roles/rest-api/defaults/main.yaml

```diff
@@ -0,0 +1,16 @@
+---
+# Git branches
+rest_api_git_branch: develop
+
+# Paths to services folder
+rest_api_workdir: /opt/rest-api
+
+# systemd unit folder
+rest_api_service_dir: /etc/systemd/system
+
+# Variables
+rest_api_postgres_host: postgres
+rest_api_postgres_port: 5433
```

Comment:
It is better to set it to 5432

Reply:
if we install all services to one host then 5432 will be owned by another service (RABBIT-TO-db)